### ISSUE TITLE
Question about Learning curves of FastSpeech.

### ISSUE THREAD

**Issue by dathudeptrai** [espnet/espnet#1890]
Do u have any learning curves (tensorboard) of fastspeech on LJSpeech dataset ?, i cann't find it in this repo.

**Comment by kan-bayashi** [comment]
Hi @dathudeptrai. I will share the training curve figure.
<img width="1908" alt="スクリーンショット 2020-04-29 午後11 18 09" src="https://user-images.githubusercontent.com/22779813/80606884-c2a7c800-8a6f-11ea-8ae2-55407643b959.png">
This is the training curve of the following model (fastspeech.v4 + taco.v3 as teacher)
https://github.com/espnet/espnet/blob/master/egs/ljspeech/tts1/RESULTS.md#models-1

**Comment by dathudeptrai** [comment]
hi @kan-bayashi seem the validation loss so high, i have similar behavior on tensorflow model now :D. BTW, l1_loss here is just mae of mel grountruth and mel_pred ?

**Comment by kan-bayashi** [comment]
Right. FastSpeech has the tendency to overfit in my experiments. This model used knowledge distillation, so the target is not ground-truth but the Mel predicted by the teacher model without teacher forcing.

**Comment by dathudeptrai** [comment]
hi @kan-bayashi , got it. I will try some new idea to avoid overfit problem now :D, i will share with u when the idea work :D.
